apps/system/lockscreen/js/state/lockscreen_basic_state.js

```diff
@@ -57,6 +61,17 @@
    * Or the component cannot track the last active state.
    */
   LockScreenBasicState.prototype.transferTo = function() {
+    if (this._transferred) {
```

Comment:
My question is simply that we shouldn't be transferring twice anyway. If so that means something is wrong with our concurrent wiring... you could certainly workaround and nullify the second call but I recommend we find the real cause.